### ISSUE TITLE
Determine Element reach

### ISSUE THREAD

**Issue by denvasden** [fisshy/react-scroll#280]
Hi!

Are there any ability to determine that the element has been reached and apply some class to this element using react-scroll?

Thanks a lot!

**Comment by fisshy** [comment]
Hey, 

Yes, theres a begin/end event. 
```
var Scroll  = require('react-scroll');
var Events     = Scroll.Events;

Events.scrollEvent.register('begin', function(to, element) {
      console.log("begin", arguments);
});

Events.scrollEvent.register('end', function(to, element) {
      console.log("end", arguments);
});
```

**Comment by denvasden** [comment]
Hi, thanks for the answer but I've tried to use this approach and nothing
happend.

I've used this code as an example.

componentDidMount() {

  Events.scrollEvent.register('begin', function() {
    console.log("begin", arguments);
  });

  Events.scrollEvent.register('end', function() {
    console.log("end", arguments);
  });

  scrollSpy.update();

}


I've used it in a component which needs to be determined and even in the
parent component but there were no console logs.

Maybe I'm using it in a wrong way?
